robosuite/robots/robot.py

```diff
@@ -536,7 +536,7 @@ def dof(self):
             int: degrees of freedom of the robot (with grippers).
         """
         # Get the dof of the base robot model
-        dof = super().dof
+        dof = self.robot_model.dof
```

Comment:
noticed we have an extra `dof` function here that needed to be removed. thanks for finding this issue!